src/test/run-pass/traits/trait-alias-import.rs

```diff
@@ -0,0 +1,23 @@
+#![feature(trait_alias)]
+
+mod inner {
+    pub trait Foo {
+        fn foo(&self);
+    }
+
+    pub struct Qux;
+
+    impl Foo for Qux {
+        fn foo(&self) {}
+    }
+
+    pub trait Bar = Foo;
+}
+
+// import only the alias, not the `Foo` trait
```

Comment:
```suggestion
// Import only the alias, not the `Foo` trait.
```
(I'm a grammar Nazi, I know...)